spec/factories/registrations.rb

```diff
@@ -38,10 +38,14 @@
       competing_status { Registrations::Helper::STATUS_WAITING_LIST }
     end
 
-    trait :newcomer do
+    trait :first_timer do
```

Comment:
I like this as a term, but I would use it the other way around!
As in, `newcomer` is someone who really has absolutely zero competition experience, while `first_timer` is someone who competed for the first time this year.

But I'm happy to defer to consensus and a broader vote